flink-streaming-java/src/main/java/org/apache/flink/streaming/api/graph/StreamingJobGraphGenerator.java

```diff
@@ -99,6 +99,10 @@
 
 	private static final int MANAGED_MEMORY_FRACTION_SCALE = 16;
 
+	private static final long DEFAULT_PIPELINED_BUFFER_TIMEOUT = 100L;
+
+	private static final long DEFAULT_BLOCKING_BUFFER_TIMEOUT = -1L;
```

Comment:
nit: rename to `UNSPECIFIED` or `ON_BUFFER_FULL`?

Reply:
I renamed `DEFAULT_PIPELINED_BUFFER_TIMEOUT` to `DEFAULT_NETWORK_BUFFER_TIMEOUT` and `DEFAULT_BLOCKING_BUFFER_TIMEOUT` to `UNDEFINED_NETWORK_BUFFER_TIMEOUT` for not involving the concept of pipelined/blocking.